src/ResourceHeader.js

```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const ResourceHeader = ({ label, index }) => {
+  return (
+    <div className="rbc-row rbc-row-resource">
+      <div key={`resource_${index}`} className="rbc-header">
+        {label}
```

Comment:
the Header component should be just for the `{label}` part otherwise specifying a component would break the styles